.github/workflows/add-new-issue-to-project.yml

```
@@ -0,0 +1,18 @@
+name: 🤖Add New Issue To Project
+
+
+on:
+  issues:
+    types: opened
+
+
+jobs:
+  add_new_issue_to_project:
+    name: Add New Issue
+    uses: KinsonDigital/Infrastructure/.github/workflows/add-issue-to-project.yml@v8.2.0
+    with:
+      org-name: "${{ vars.ORGANIZATION_NAME }}"
+      org-project-name: "${{ vars.ORG_PROJECT_NAME }}"
+      project-name: "${{ vars.PROJECT_NAME }}"
+    secrets:
+      cicd-pat: ${{ secrets.CICD_REST_API }}
```

Update token reference:

```suggestion
      cicd-pat: ${{ secrets.CICD_ TOKEN }}
```